### ISSUE TITLE
Change æ in italic

### ISSUE THREAD

**Issue by sigurdhasle** [indestructible-type/Besley#34]
The a (and å) are two-storied in roman and change to single-story in italic. Excellent! But I think the a-part of the æ should change in the same way – like in this random clarendonesque font I found on the internet. (Please put this on the to-do-list _after_ the small caps Æ!)
![image](https://user-images.githubusercontent.com/47284316/211554035-deed133f-1b2e-4643-8164-fc7832e23ff3.png)

**Comment by ctrlcctrlv** [comment]
That's not universal, it's a design decision.

The little bit sticking up is ugly IMO and hard to deal with at heavy weights, I usually avoid doing that in my fonts.

Here's another random font that does it as does Besley:

![image](https://user-images.githubusercontent.com/838783/211626653-cb230513-8e2b-4bf4-8fe9-aa9cdaa16c3d.png)

But ultimately up to Owen

**Comment by sigurdhasle** [comment]
You are right, it is a decision. I my opinion it should change in the same way as the a's, and I do not think the little bit is ugly – that may be because i see a lot of æ's all the time (it is in my native language).

On the other hand: it is not my decision,either, just a suggestion.
